tests/tests/test_configs.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+import jsonschema
+
+from zarr_benchmarks import utils
+
+
+def test_schemas_valid():
+    """Validate all jsons in the 'benchmarks_configs' dir against the schema"""
+
+    configs_dir = Path("tests/benchmarks/benchmark_configs")
```

Comment:
```suggestion
    configs_dir = Path(__file__).parent.parent / "benchmarks" / "benchmark_configs"
```
Probably best to use a relative path to the directory, just in case this file is imported from a different working directory.